修正使用Python3作为默认环境时，在中文操作系统中无法正常安装的BUG
diff -urN Pymacs-master/pppp Pymacs-master.orig/pppp
--- Pymacs-master/pppp	2014-05-28 10:52:14 +0800
+++ Pymacs-master.orig/pppp	2014-03-02 04:41:04 +0800
@@ -241,7 +241,7 @@
                 if self.verbose:
                     sys.stderr.write("writing %s\n" % output)
                 self.transform_file(
-                    input, open(input, encoding='utf-8'), open(output, 'w', encoding='utf-8').write)
+                    input, open(input), open(output, 'w').write)
                 if output.endswith('.py'):
                     pyc_file = output[:-2] + '.pyc'
                     if os.path.exists(pyc_file):
